### PR TITLE
Controller spec

### PR DESCRIPTION
**Pull request by Elijahdre**
In this PR, I have :
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  describe 'GET /show' do
+    it 'returns http success and check correct placeholder text.' do
+      get '/users/:author_id/posts/:id'
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to include('Here is a specific post')
+    end
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline spec/requests/post_spec.rb]: - [ ] kindly add a test that ensures correct view template is rendered.
Example :` expect(response).to render_template(:show)`

File: spec/requests/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  describe 'GET /show' do
+    it 'returns http success and check correct placeholder text.' do
+      get '/users/:id'
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to include('Here is a specific user')
+    end
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline spec/requests/user_spec.rb]: - [ ] kindly add a test that ensures the correct view template is rendered.
Example :` expect(response).to render_template(:show)`

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Elijahdre  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — APPROVED** [review]
Hi @Elijahdre, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**